### PR TITLE
new Galaxy wrappers for version 2.0

### PR DESCRIPTION
**Pull request by bgruening**
depends on https://github.com/fidelram/deepTools/issues/144

### CODE REVIEW

**Comment by bgruening** [comment]
@fidelram, @dpryan79 here is a package with the latest 1.6 branch: https://testtoolshed.g2.bx.psu.edu/view/iuc/package_python_2_7_deeptools_2_0
If this PR gets merged we just need to rebuild the container and get always the latest 1.6 branch with all bugfixes. As soon as we tag a 2.0 release we simply change the URL in the tool_dependencies.xml file and we should be ready.

**Comment by dpryan79** [comment]
Can you remove emf from the list of image formats? Matplotlib doesn't actually support it in Galaxy (not that anyone uses it anyway).

**Comment by bgruening** [comment]
Done!

**Comment by dpryan79** [comment]
LGTM!

**Comment by bgruening** [comment]
Great, I will wait until the blocking issue is fixed, this will streamline the handling of tools.

**Comment by fidelram** [comment]
@bgruening #144 has been addressed, now --keepNAs is in place. Also the --MNAse parameter of bamCoverage is working.

**Comment by bgruening** [comment]
Out of WIP, at least as a first version.

**Comment by dpryan79** [comment]
Cool! We have a local instance of this now, so I'll look at the live versions tomorrow and start testing :+1:

File: galaxy/wrapper/deepTools_macros.xml [hunk 1]
```diff
@@ -308,23 +380,25 @@ This tool is developed by the `Bioinformatics and Deep-Sequencing Unit`_ at the
...
-            <option value="emf">emf</option>
         </param>
     </xml>
 
-    <xml name="missingDataAsZero">
-                <param name="missingDataAsZero" type="boolean" truevalue="yes" falsevalue="no" checked="True"
-                    label ="Treat missing data as zero"
-                    help  ="This parameter determines if missing data should be treated as zeros. If unchecked, missing data will be ignored and not included in the output file. Missing data is defined as those regions for which both BAM files have 0 reads." />
+    <xml name="keepNAs">
+        <param argument="--keepNAs" type="boolean" truevalue="--keepNAs" falsevalue="" checked="False"
```
Comment (fidelram) [inline galaxy/wrapper/deepTools_macros.xml]: @bgruening we renamed this to be --skipNonCoveredRegions (or simply --skipNAs). 

The help says:

 'This parameter determines if non covered regions regions without overlapping reads) in a bam file should 
  be skipped. The default is to treat those regions as having a value of zero.  The decision to skip non 
 covered regions depends on the interpretation of the data. Non covered regions may represent for 
 example repetitive regions that want to be skipped.

**Pull request merged**